app/_locales/en/messages.json

```diff
@@ -4141,6 +4141,9 @@
   "someNetworksMayPoseSecurity": {
     "message": "Some networks may pose security and/or privacy risks. Understand the risks before adding & using a network."
   },
+  "somethingDoesntLookRight": {
+    "message": "Something doesn't look right? $1"
```

Comment:
missing "description" for $1

hmm I thought I remembered a linter catching this in the past. maybe it was a comment in my own PR.